src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -46,7 +46,9 @@
          [quo/wallet-graph {:time-frame :empty}]
          (when (not watch-only?)
            [quo/wallet-ctas
-            {:send-action    #(rf/dispatch [:open-modal :wallet-select-address])
+            {:send-action    (fn []
+                               (rf/dispatch [:wallet/clean-send-data])
```

Comment:
@ulisesmac, @briansztamfater I ran into some issues cleaning send data on navigation away from the transaction progress page. it causes a stack overflow. Anyway I found that is probably safer to add the clean event to the start of the flow so I added it here.
wdyt? 🤔